packages/manager/modules/overthebox/src/overthebox/actions/index.js

```diff
@@ -0,0 +1,24 @@
+import angular from 'angular';
+import controller from './overTheBox-actions.controller';
+import template from './overTheBox-actions.html';
+
+const moduleName = 'ovhManagerOtbActions';
+
+angular.module(moduleName, []).config(($stateProvider) => {
+  $stateProvider.state('overTheBoxes.overTheBox.actions', {
+    url: '/actions',
+    views: {
+      otbView: {
+        template,
+        controller,
+        controllerAs: '$ctrl',
+      },
+    },
+    translations: {
```

Comment:
You can use:
```
.run(/* @ngTranslationsInject:json ./translations */)
```
for loading translations. Your translations will be therefore linked to your module and not to the route ;-)
Don't forget to import and add as dependency to your module `@ovh-ux/ng-translate-async-loader` and `angular-translate`